packages/api/src/services/emp-state.ts

```diff
@@ -145,6 +148,21 @@ export default (config: Config, appState: Dependencies) => {
       }
       // add any new sponsors
       await emps.active.addSponsors(address, eventState.sponsors || []);
+
+      // set created timestamp if needed
+      await updateCreatedTimestamp(address, emps.active);
+    }
+  }
+
+  async function updateCreatedTimestamp(address: string, table: uma.tables.emps.JsMap) {
+    if (registeredEmpsMetadata.has(address)) {
```

Comment:
this is just personal preference of mine, but i like to write guards that exit early rather than nest.
for instance 
```js
if(!registeredEmpsMetadata.has(address)) return
const emp = await table.get(address);
const blockMetadata = registeredEmpsMetadata.get(address);
if (emp.createdTimestamp) return
```

This is not strictly enforced so its up to you, but i find the flatter code a bit cleaner

Reply:
I totally agree with you. Sometimes I write guards, sometimes I do it using if else statements. I'll stick with guards from now